src/node_crypto.cc

```diff
@@ -2773,6 +2775,38 @@ void SSLWrap<Base>::GetSharedSigalgs(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+template <class Base>
+void SSLWrap<Base>::ExportKeyingMaterial(
+                                      const FunctionCallbackInfo<Value>& args
+                                      ) {
+  CHECK(args[0]->IsInt32());
+  CHECK(args[1]->IsString());
+
+  Base* w;
+  ASSIGN_OR_RETURN_UNWRAP(&w, args.Holder());
+  Environment* env = w->ssl_env();
+
+  uint32_t olen = args[0].As<Uint32>()->Value();
+  node::Utf8Value label(env->isolate(), args[1].As<String>());
+
+  AllocatedBuffer out = env->AllocateManaged(olen);
+
+  if (SSL_export_keying_material(w->ssl_.get(),
+              reinterpret_cast<unsigned char*>(out.data()),
+              olen,
+              *label,
+              label.length(),
+              nullptr,  // the context parameter is optional
+              0,
+              0) != 1) {
+    return env->ThrowError(
+      "Export Keying Material returned invalid status code");
```

Comment:
```suggestion
        "Export Keying Material returned invalid status code");
```

Reply:
just out of curiosity, is there any auto linter for the c++ code / javascript code in place?

Reply:
@simllll You can use something like `CLANG_FORMAT_START=master make format-cpp` to use clang-format to format your code, if you like :)

Reply:
thanks @addaleax  :)